advanced_source/static_quantization_tutorial.py

```diff
@@ -1,28 +1,24 @@
 """
-(experimental) Static Quantization with Eager Mode in PyTorch
+(실험용) PyTorch에서 eager mode로 실행하는 정적 양자화 
 =========================================================
 
 **Author**: `Raghuraman Krishnamoorthi <https://github.com/raghuramank100>`_
 
 **Edited by**: `Seth Weidman <https://github.com/SethHWeidman/>`_
 
-This tutorial shows how to do post-training static quantization, as well as illustrating
-two more advanced techniques - per-channel quantization and quantization-aware training -
-to further improve the model's accuracy. Note that quantization is currently only supported
-for CPUs, so we will not be utilizing GPUs / CUDA in this tutorial.
+**번역**: `Choi Yoonjeong <https://github.com/potatochips178/>`_
 
-By the end of this tutorial, you will see how quantization in PyTorch can result in
-significant decreases in model size while increasing speed. Furthermore, you'll see how
-to easily apply some advanced quantization techniques shown
-`here <https://arxiv.org/abs/1806.08342>`_ so that your quantized models take much less
-of an accuracy hit than they would otherwise.
 
-Warning: we use a lot of boilerplate code from other PyTorch repos to, for example,
-define the ``MobileNetV2`` model archtecture, define data loaders, and so on. We of course
-encourage you to read it; but if you want to get to the quantization features, feel free
-to skip to the "4. Post-training static quantization" section.
+이 튜토리얼은 모델의 정확성을 더욱 높이기 위한 두가지 발전된 기술 - 채널별 양자화(per-channel quantization)와 양자화 자각 훈련(quantization-aware training)- 을 설명할 뿐 아니라 어떻게 훈련 후 정적 양자화(post-training static quantization)가 동작하는지 보여줍니다. 
+최근 양자화는 CPU를 통해서만 사용 가능하기 때문에 GPU/CUDA를 사용하지 않을 것을 알립니다. 
 
-We'll start by doing the necessary imports:
+이 튜토리얼의 마지막에서는 PyTorch에서 양자화가 어떻게 속도를 오르게 하는 반면 모델 크기를 줄이는 결과가 나오는지 볼 수 있습니다. 
+게다가 `여기서 <https://arxiv.org/abs/1806.08342>`_ 보인 발전된 양자화 기술들을 쉽게 적용시키는 법을 볼 수 있을 것이며 양자화된 모델들은 그렇지 않은 모델보다 훨씬 더 적은 정확성 일치를 가집니다. 
+
+주의: ``MobileNetV2`` 모델 구조 정의하기, 데이터 로더 정의하기 등과 같은 다른 PyTorch 레포지토리에서 공통으로 사용되는 많은 코드를 사용할 예정입니다. 
+물론 사용자들이 이것을 읽기 바랍니다. 그러나 만약 양자화 특징을 얻길 원한다면 자유롭게 건너뛰고 "4. 훈련 후 정적 양자화" 부분으로 넘어가면 됩니다. 
```

Comment:
```suggestion
처음부터 읽으면 좋지만 양자화 특징에 대해 알고싶다면 "4. 훈련 후 정적 양자화" 부분으로 넘어가면 됩니다. 
```

의역을 넣어 좀더 매끄럽게 만들어보았습니다.